frontend/src/utilities/notebookControllerUtils.ts

```diff
@@ -202,7 +202,8 @@ export const checkNotebookRunning = (notebook?: Notebook): boolean =>
   !!(
     notebook?.status?.readyReplicas &&
     notebook?.status?.readyReplicas >= 1 &&
-    notebook?.metadata.annotations?.['opendatahub.io/link']
+    notebook?.metadata.annotations?.['opendatahub.io/link'] &&
+    !notebook?.metadata.annotations?.['kubeflow-resource-stopped']
```

Comment:
love this function -- worked out great to align everyone on the new state of what is a running notebook and what isn't. Thanks for creating it @DaoDaoNoCode